src/gh_issues/__init__.py

```diff
@@ -31,18 +33,20 @@ def issue_fields(self) -> typing.Generator[str, None, None]:
     @property
     def title(self) -> str:
         return self._title
-    
-    def get_content_issues(self, field_name: str) -> list['Issue']:
+
+    def get_content_issues(self, field_name: str) -> list["Issue"]:
         for issue in get_content_issues(getattr(self, field_name)):
             yield type(self).from_issue_number(self._repo, issue)
 
     @classmethod
-    def from_issue_number(cls, repo: Repo, issue_id: int, api_token: str | None=None) -> 'Issue':
+    def from_issue_number(
+        cls, repo: Repo, issue_id: int, api_token: str | None = None
```

Comment:
This is great! Would you mind changing `issue_id` here as well otherwise the tests won't pass

Reply:
```suggestion
        cls, repo: Repo, issue_number: int, api_token: str | None = None
```

Reply:
Thanks for the gentle reminder to run the tests in addition to the pre-commit before making a pr.

Reply:
I think technically the tests would have passed, but the change would have been on the internal portions of the code. Not the stuff that users interact with.